db/queries/insertTimezone.js

```diff
@@ -0,0 +1,13 @@
+const db = require('../index.js');
+
+module.exports = function insertTimezone(values) {
+  return db.query(
+    "INSERT INTO user_timezones(username, username_discriminator, timezone) " +
+    "VALUES($1, $2, $3) " +
+    "ON CONFLICT (username_discriminator) DO UPDATE " +
+    "SET timezone = $3",
+    values
+  )
+  .then(res => {return res.rowCount;})
+  .catch(err => console.error(err.stack));

```

Comment:
why console log error instead of letting it persist? remote servers arent going to be able to do anything with the stdout and no ones ever going to see it

Reply:
I was considering adding a logger eventually. For now, consoles, unless you feel that strongly against it.

Reply:
nope. if there's plans for a logger that's fine. node apps blow on exceptions unlike other frameworks so it makes sense to catch before you get the logger going